Сhapter4/Chapter4.Task6/Program.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+//6. створити двовимірний масив, розміром n на m(n - кіслькість рядків, m - кількість стовпців). n та m не обовязково зчитувати з консолі.Заповнити довільними значеннями матрицю.Створити два одновимірні масиви. Перший, розміром n, кожен елемент якого - це сума чисел чисел у рядку. другий, розміром m, кожен елемент якого - це сума чисел чисел у стовпці
+
+namespace Chapter4.Task6
+{
+    class Program
+    {
+        static int ReadInt(string text)
+        {
+            Console.WriteLine(text);
+            string numStr = Console.ReadLine();
+            int num = int.Parse(numStr);
+
+            return num;
+        }
+        static void WriteMatrix(int[,] matrix, int numRows, int numColums)
+        {
+            for (int i = 0; i < numRows; i++)
+            {
+                for (int j = 0; j < numColums; j++)
+                {
+                    Console.Write("{0 } ", matrix[i, j]);
+                }
+                Console.WriteLine();
+            }
+        }
+        static void WriteArrays(int [] Array,  int numRows)
```

Comment:
method name confuses: WriteArrays - means that you writes here more than one array. Rename it
numRows is not suitable here. You can get known of array length via it one of properties.